src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -79,19 +79,19 @@ def create_configstore(cmd,
     config_store = client.begin_create(resource_group_name, name, configstore_params)
 
     # # Poll request and create replica after store is created
-    while config_store.status() == ProvisioningStatus.RUNNING:
+    while config_store.status() != ProvisioningStatus.SUCCEEDED:
         progress.spinner.step(label="Creating store")
-        config_store.wait(3)
+        config_store.wait(1)
 
     if config_store.status() == ProvisioningStatus.SUCCEEDED and replica_name is not None:
```

Comment:
nit: We do not need to check `config_store.status() == ProvisioninsStatus.SUCCEEDED`

Reply:
Yeah, updated